catalogue/webapp/components/IIIFImage/IIIFImage.tsx

```diff
@@ -33,43 +33,59 @@ const IIIFImage: FC<Props> = ({
   onLoadingComplete,
   priority = false,
   layout,
-  width = 300,
 }) => {
   const sizesString = sizes
     ? convertBreakpointSizesToSizes(sizes).join(', ')
     : undefined;
 
+  // The Nextjs Image component has an experimental 'raw' layout feature
+  // which will be rendered as a single image element with no wrappers, sizers or other responsive behavior.
+  // We may be able to use this in future but, until then, render our own img element.
+  if (layout === 'raw') {
+    return (
+      <img
+        src={iiifImageTemplate(image.contentUrl)({
+          size: `${image.width},`,
+        })}
+        srcSet={''}
+        sizes={sizesString}
+      />
+    );
+  }
+
+  if (layout === 'fixed') {
+    return (
+      <Image
+        layout={layout}
+        sizes={sizesString}
+        src={image.contentUrl}
+        alt={image.alt || ''}
+        loader={IIIFLoader}
+        onLoadingComplete={onLoadingComplete}
+        width={image.width}
+        height={image.height}
+        priority={priority}
+        placeholder="blur"
+        blurDataURL="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNctWxZEQAGYgJqdIxNSwAAAABJRU5ErkJggg=="
+      />
+    );
+  }
+
+  // {/* TODO: is fill layout still needed? */}
```

Comment:
Dunno, try deleting it and see what happens. 😛

Reply:
Keeping it, it's used elsewhere (in works) and has a different aim than the others!